index.html

```diff
@@ -35,12 +35,144 @@ <h3>I am a Software Developer</h3>
             <ul>
                 <li><a href="#"><i class="bi bi-github"></i></a></li>
                 <li><a href="#"><i class="bi bi-linkedin"></i></a></li>
-                <li><a href="#"><img src="NormalButton/Tertiary/Icons/Vector.svg" alt="peace button social media"></a>
+                <li><a href="#"><img src="vectorsImg/Tertiary/Icons/Vector.svg" alt="peace button social media"></a>
                 </li>
                 <li><a href="#"><i class="bi bi-twitter"></i></a></li>
-                <li><a href="#"><img src="NormalButton/Tertiary/Icons/Mvector.svg" alt="m button social media"></a></li>
+                <li><a href="#"><img src="vectorsImg/Tertiary/Icons/Mvector.svg" alt="m button social media"></a></li>
             </ul>
         </section>
+
+        <section class="work-section">
+            <h2>My Recent Works </h2>
+
+            <article class="card">
+                <img src="images/ImgProj.png" alt="Project image">
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.</p>
+                <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#" class="first-btn">See Project</a>
+            </article>
+
+            <article class="card card-bg">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                <a href="#" class="fullWidth-btn">See Project</a>
+            </article>
+
+            <article class="card card-bg">
+                <div class="padding-card">
+                    <h3>Profesional Art Printing Data</h3>
```

Comment:
- [ ] Please make sure to use the same font family used for the button text here

![image](https://user-images.githubusercontent.com/74744787/164720038-df79e78d-ab7e-49de-866e-82882e3dc6a5.png)

I noticed that you have been using Poppins here instead of inter